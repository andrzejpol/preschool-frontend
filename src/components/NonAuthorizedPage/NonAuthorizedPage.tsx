import React from "react";

const NotAuthorizedPage = () => {
  return (
    <div>
      <h1>403 - Not Authorized</h1>
      <p>You do not have permission to access this page.</p>
    </div>
  );
};

export default NotAuthorizedPage;