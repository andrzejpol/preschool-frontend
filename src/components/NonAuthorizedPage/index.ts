import NonAuthorizedPage from './NonAuthorizedPage';

export default NonAuthorizedPage;