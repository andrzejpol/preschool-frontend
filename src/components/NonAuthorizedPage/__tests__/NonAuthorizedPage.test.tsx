import React from "react";
import {render} from "@testing-library/react";
import NonAuthorizedPage from "../NonAuthorizedPage";

describe("NotAuthorizedPage Component", () => {
  it("matches the snapshot", () => {
    const {asFragment} = render(<NonAuthorizedPage/>);

    // Snapshot assertion
    expect(asFragment()).toMatchSnapshot();
  });
});
