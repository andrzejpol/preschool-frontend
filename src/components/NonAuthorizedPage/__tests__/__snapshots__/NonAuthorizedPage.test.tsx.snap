// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`NotAuthorizedPage Component > matches the snapshot 1`] = `
<DocumentFragment>
  <div>
    <h1>
      403 - Not Authorized
    </h1>
    <p>
      You do not have permission to access this page.
    </p>
  </div>
</DocumentFragment>
`;
